vendor/knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake/fake.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
how this file added? 
is there any changes in go.mod file

Reply:
Its package is already under go.mod.  Changes are in vendor/modules.txt file.

Reply:
Its due to the new import here: 	fakepodinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake"